components/google-cloud/google_cloud_componets/mb_sdk_component/utils.py

```diff
@@ -0,0 +1,98 @@
+# Lint as: python3
```

Comment:
The directory is misspelled for these files.

Reply:
good catch! corrected the directory name.